src/components/tutorial-meta/components/variant-list/variant-list.module.css

```diff
@@ -1,13 +1,3 @@
-.root {
```

Comment:
I moved the layout styles to the parent component stylesheet to control the hide / show of the dropdown versus list, felt like a better place than within this component.